src/app/common/hooks/balance/stx/use-stx-balance.ts

```diff
@@ -26,6 +26,13 @@ export function useStxBalance() {
     ? i18nFormatCurrency(baseCurrencyAmountInQuote(stxEffectiveBalance, stxMarketData))
     : undefined;
 
+  const stxLockedBalance = anchoredBalanceQuery.data?.stx.locked;
+  const stxUsdLockedBalance = isDefined(stxLockedBalance)
+    ? i18nFormatCurrency(
+        baseCurrencyAmountInQuote({ ...stxLockedBalance, decimals: 0 }, stxMarketData)
```

Comment:
I don't follow what's happening here. STX locked balance is being passed in. STX has 6 decimals, so afaict this would yield the wrong number?

Reply:
fixed it in `parseBalanceResponse` function